unity-renderer/Assets/DCLPlugins/ECS7/Systems/ECSSystemsController.cs

```diff
@@ -32,6 +35,11 @@ public ECSSystemsController(ECS7System componentWriteSystem, SystemsContext cont
         hoverCanvas.name = "_ECSInteractionHoverCanvas";
         IECSInteractionHoverCanvas interactionHoverCanvas = hoverCanvas.GetComponent<IECSInteractionHoverCanvas>();
 
+        var scenesUiResource = Resources.Load<UIDocument>("ScenesUI");
```

Comment:
you can make use of better dependency injection here:
- Environment.i
- DataStore.i
- ECSInteractionHoverCanvas object
-ScenesUI object